src/Microsoft.Identity.Client/TokenCache.cs

```diff
@@ -816,6 +817,7 @@ async Task<IEnumerable<IAccount>> ITokenCacheInternal.GetAccountsAsync(string au
             {
                 foreach (MsalAccountCacheItem account in accountCacheItems)
                 {
+                    // todo: look at usage of homeaccountid if it's null....
                     if (RtMatchesAccount(rtItem, account))
```

Comment:
Is this something that should be done now? definitely open an issue or put this somewhere so we remember to look into it. same w/one on line 844.